proto/zitadel/management.proto

```diff
@@ -841,6 +841,76 @@ service ManagementService {
         };
     }
 
+    // Sets a org metadata by key
+    rpc SetOrgMetadata(SetOrgMetadataRequest) returns (SetOrgMetadataResponse) {
+        option (google.api.http) = {
+            post: "/orgs/me/metadata/{key}"
```

Comment:
just discussd this with livio. we shoul make the requests /metadata instead of orgs/me/metadata

I think this is something we had in the begining and it doesn't make a lot of sense. so we try to do it better with the new implemented requests. 

Please change it for all the requests